lib/ExpensiMark.js

```diff
@@ -250,4 +255,71 @@ export default class ExpensiMark {
 
         return replacedText;
     }
+
+    /**
+     * Modify text for Quotes replacing chevrons with html elements
+     *
+     * @param {RegExp} regex
+     * @param {String} textToCheck
+     *
+     * @returns {String}
+     */
+
+    modifyTextForQuote(regex, textToCheck) {
+        let replacedText = '';
+        let TextToFormat = '';
+        const match = textToCheck.match(regex);
+
+        // if there's matches we need to modify the quotes
+        if (match !== null) {
+            // split the textToCheck in lines
+            const textSplitted = textToCheck.split('\n');
+            for (let i = 0; i < textSplitted.length; i++) {
+                // we only want to modify lines starting with &gt;
+                if (textSplitted[i].substr(0, 4) === '&gt;') {
+                    // avoid blank lines starting with &gt;
+                    if (textSplitted[i].trim() !== '&gt;') {
+                        textSplitted[i] = textSplitted[i].substr(4, textSplitted[i].length);
+                        textSplitted[i] = textSplitted[i].trim();
+                        TextToFormat += `${textSplitted[i]}\n`;
+                    } else {
+                        // we need ensure that index i-1 >= 0
+                        if (i > 0) {
```

Comment:
Ah yeah this is a simpler error, you should probably have this inside an `else if()` instead of an `else { if () {`. Does that make sense?

Reply:
yes